plugins/grib_pi/src/CustomGrid.cpp

```diff
@@ -21,9 +15,7 @@
  *   along with this program; if not, write to the                         *
  *   Free Software Foundation, Inc.,                                       *
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
- ***************************************************************************
- *
- */
+ ***************************************************************************/
```

Comment:
Likewise,  here you dropped the Purpose line without adding any file. At a minimum, just add

```
/**
* \file
* Personalized GRID header
* /
```
just to not lose information. This applies to all headers where you removed a non-empty Purpose line without adding a \file section.

Reply:
Done in every .cpp file.
Improved the description for CustomGrid.h